stories/Badge.stories.tsx

```diff
@@ -11,38 +11,46 @@ export const Default = Template.bind({});
 
 Default.args = {
   children: 'Some Text',
-  variant: ''
 };
 
+export const Positive: React.FC = () => (
+  <>
+    <Badge variant={BadgeType.POSITIVE} className="tk-mr-h">Badge</Badge>
```

Comment:
Did we change this because it doesn't work anymore as it was before?

Reply:
it's works as well with the string, you just get an eslint warning.
I thought it was better with an enum, also to stick the Banner component implementation, but I can revert if you don't like.